src/Sylius/Component/Order/Model/Order.php

```diff
@@ -357,8 +337,6 @@ public function removeAdjustment(AdjustmentInterface $adjustment)
             $adjustment->setAdjustable(null);
```

Comment:
This should be removed as item __must__ be assigned to some "parent", otherwise it's broken.